examples/analog/streaming_synchronized.cpp

```diff
@@ -0,0 +1,63 @@
+#include <cmath>
+#include <libm2k/m2k.hpp>
+#include <libm2k/contextbuilder.hpp>
+#include <libm2k/analog/m2kanalogout.hpp>
+#include <thread>
+
+using namespace std;
+using namespace libm2k;
+using namespace libm2k::analog;
+using namespace libm2k::contexts;
+
+constexpr float pi = 3.14;
+constexpr double sampleRate = 75000;
+
+int main()
+{
+	M2k *context = m2kOpen("usb:5.15.5");
```

Comment:
I think we should remove the uri from this example.

Reply:
Removed in commit 6205d71.